noggin/templates/_form_macros.html

```diff
@@ -15,7 +15,7 @@
   {% endif %}
   {{ field(class=css_class, **kwargs) }}
   {% if field.type == 'BooleanField' %}
-      <label class="form-check-label ml-1"><strong>{{ field.label.text }}</strong></label>
+    <label class="form-check-label ml-1" {% if field.id %}for="{{ field.id }}"{% endif %}><strong>{{ field.label.text }}</strong></label>
```

Comment:
I don't think you need the test here, fields of forms built with WTForms always have an ID.